src/lib/y2storage/callbacks/probe.rb

```diff
@@ -110,6 +125,53 @@ def again?
 
       private
 
+      def details(what)
+        return what unless what.match?(/WARNING: PV .* was already found on .*/i)
```

Comment:
Reminder: this is going to be changed. See https://github.com/yast/yast-storage-ng/pull/1247, https://github.com/yast/yast-storage-ng/pull/1249, and https://github.com/yast/yast-storage-ng/pull/1250. The same for SP4/master is on the way.

Reply:
PR for master is now ready https://github.com/yast/yast-storage-ng/pull/1251